static/js/publisher/market/markdown.js

```diff
@@ -0,0 +1,14 @@
+export default function() {
+  const markdownLink = document.querySelector(".js-toggle-markdown");
+  const markdownInfo = document.querySelector(".js-markdown");
+
+  markdownLink.addEventListener("click", e => {
+    e.preventDefault();
+    markdownInfo.classList.toggle("u-hide");
+    if (markdownInfo.classList.contains("u-hide")) {
+      markdownLink.innerHTML = markdownLink.innerHTML.replace("Hide", "Show");
+    } else {
+      markdownLink.innerHTML = markdownLink.innerHTML.replace("Show", "Hide");
```

Comment:
This solution heavily relies on the HTML content and the structure of the link label. Someone later may change the "Show supported markdown syntax" to something else and this will silently stop working.

Would be better if this would set whole link label to "Show supported ..." / "Hide supported ...", so at least when searching project for "Show supported markdown" it would show up.

Or - ideally - don't include it in HTML template at all, just render in JS. But that would be better if the page was rendering in React. I won't mind if we skip this here for sake of this PR.